cleanlab/regression/rank.py

```diff
@@ -0,0 +1,150 @@
+import numpy as np
+from cleanlab.outlier import OutOfDistribution
+from sklearn.neighbors import NearestNeighbors
+from cleanlab.internal.regression_utils import assert_valid_inputs
+from typing import Dict, Callable
+
+""" Generates label quality scores for every sample in regression dataset """
+
+
+def get_label_quality_scores(
+    labels: np.ndarray,
+    predictions: np.ndarray,
+    *,
+    method: str = "TO_BE_NAMED",  # TODO update name once finalised
+) -> np.ndarray:
+    """
+    Returns label quality score for each example in the regression dataset.
+
+    Each score is a continous value in the range [0,1]
+    1 - clean label (given label is likely correct).
+    0 - dirty label (given label is likely incorrect).
+
+    Parameters
+    ----------
+    labels : np.ndarray
+        Raw labels from original dataset.
+        Array of shape ``(N, )`` consisting given labels, where N is number of datapoints in the regression dataset.
```

Comment:
```suggestion
        1D array of shape ``(N, )`` containing the given labels for each example (aka. Y-value, response, target, dependent variable, ...), where N is number of examples in the dataset.
```

Reply:
we use "examples" mostly instead of "datapoints"

Reply:
[Apply Docstring suggestions from code review](https://github.com/cleanlab/cleanlab/pull/572/commits/305d11bd271d29001e02d757d140bad91a749284)